### PR TITLE
Susan/update bulk upload

### PR DESCRIPTION
**Pull request by sadelokiki**
Clean up UI view for bulk upload

### CODE REVIEW

File: app/assets/javascripts/components/bulkUploadImport.jsx [hunk 1]
```diff
@@ -42,24 +43,46 @@ class BulkUploadImport extends React.Component {
...
+    $(".checkAll").click((e) => {
+      checkedStatus = e.target.checked;
+      this.setState({
+        allChecked: checkedStatus
+      }, () => {
+        $('input:checkbox').not(this).prop('checked', this.state.allChecked);
+        $(".sample-box").each((id, element) => {
+            let sampleList = this.state.selectedSampleIndices
+            if (this.state.allChecked) {
+              sampleList.push(+id);
```
Comment (yunfangjuan) [inline app/assets/javascripts/components/bulkUploadImport.jsx]: this can be a problem when sampleIndices is non-empty in the beginning

**Comment by yunfangjuan** [comment]
can you disable the submit button once i clicked on it. It should prolly say submitting when doing it.

**Comment by boris-dimitrov** [comment]
Pasting here @yunfangjuan's comment that was posted accidentally on a different pull request.

"can you disable the submit button once i clicked on it. It should prolly say submitting when doing it."

**Comment by yunfangjuan** [comment]
cool~

**Pull request merged**